deps.bzl

```diff
@@ -1,8 +1,8 @@
 load("@bazel_gazelle//:deps.bzl", "go_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
 
-# bazelisk run //:gazelle -- update-repos -from_file=go.mod -to_macro=deps.bzl%install_buildbuddy_dependencies
-def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
+# bazelisk run //:gazelle -- update-repos -from_file=go.mod -to_macro=deps.bzl%install_go_mod_dependencies
+def install_go_mod_dependencies(workspace_name = "buildbuddy"):
```

Comment:
I think there are some http_archives and http_files that we might wanna put in a go_static_dependencies target or something if this is gonna be go_mod only

Reply:
Pulled those out into their own function - there's no harm in having them together, but this makes it clearer which ones are generated and which ones are manual.